src/components/HomePageHeader/styles.module.css

```diff
@@ -0,0 +1,24 @@
+:root {
+  --fty-hero-text: #ededed;
+  --fty-hero-color-filter: 0;
+}
+
+[data-theme='dark'] {
+  --fty-hero-text: #ededed;
+  --fty-hero-color-filter: 1;
+}
+
+.hero {
+  background: url('/static/img/bg-pattern-dark.png');
```

Comment:
we can't use absolute URL to `https://www.fastify.io/images/bg-pattern-dark.3f8cda863b3bbf52.png`

I have included the image